hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/policies/router/TestLoadBasedRouterPolicy.java

```diff
@@ -131,7 +133,7 @@ public void testIfNoSubclustersWithWeightOne() {
       fail();
     } catch (YarnException ex) {
       Assert.assertTrue(
```

Comment:
LambdaTestUtils?

Reply:
Thanks for your suggestion, I will modify the code.